src/qml/Statusbar.qml

```diff
@@ -38,7 +38,7 @@ import MeeGo.Connman 0.2
 Item {
     id: root
     z: 201
-    height: 48
+    height: root.width/12
```

Comment:
Ok thanks.:) Yeah that is the problem I don't myself have low resolution device to test with.